proc/proc_windows.go

```diff
@@ -26,7 +26,7 @@ type OSProcessDetails struct {
 }
 
 // Launch creates and begins debugging a new process.
-func Launch(cmd []string) (*Process, error) {
+func Launch(cmd []string, wd string) (*Process, error) {
```

Comment:
@alexbrainman A quick grep through go source suggest no way to pass DEBUG_ONLY_THIS_PROCESS to a newly created process, if os/exec used syscall.SysProcAttr.Ptrace to set DEBUG_ONLY_THIS_PROCESS on windows we could probably use it. We do use it on linux.